port/linux/omrvmem.c

```diff
@@ -1081,6 +1087,7 @@ default_pageSize_reserve_memory(struct OMRPortLibrary *portLibrary, void *addres
 	int flags = 0;
 	void *result = NULL;
 	int protectionFlags = PROT_NONE;
+	uintptr_t allocator = 0;
```

Comment:
Minor: 'Variables should always be declared in the innermost possible scope'
https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md